pom.xml

```diff
@@ -689,11 +690,9 @@
           <systemProperties>
             <property>
               <name>hudson.udp</name>
-              <value>33849</value>
+              <value>-1</value>
```

Comment:
I mean why would we want this... like EVER? (if some esoteric plugin needs it for a test then let them configure it?)

Reply:
UDP disabling without throwing errors was only added in 1.514: https://github.com/jenkinsci/jenkins/commit/19f34b54d6cd69f837552a21d8adb41f3b5fba91

This option is set at least since 1.388: https://github.com/jenkinsci/jenkins/blob/ea06f60c1abe408ccb45c700421c0456ce00ef6d/plugins/pom.xml#L196, probably a lot longer.

Reply:
@daniel-beck that's a novel use of "only" - something that happened more than 4 years ago :-)

do you care about that supporting building plugins for something as old as that?

Reply:
I'm just answering your comment. This didn't disable, because disabling didn't exist.

Reply:
Fine, though better to just move the code out of core, as is already filed in a `split-plugins-from-core` issue.